### DISCUSSION TITLE
Plans to support RISC-V Esp32

### DISCUSSION THREAD

**Discussion by alberk8** [esphome/esp-stacktrace-decoder#3] [General]
Hi,
I really like your esp backtrace decoder for ESP32.  May I ask if there are plans to support the RISC-V version of ESP32 like the C3,C6, etc

Thank you.

**Comment by maximeborges** [comment]
Since this code is just matching stack trace addresses to the debug info from the .elf file, I would expect it to work with different architectures. Do you have any example of a stack trace from any RISC-V MCU and corresponding .elf file?

**Reply by alberk8** [reply]
I am sure that the output of the backtrack for the RISC-V is different from the Xtensa.   At the moment I do not have the backtrace handy.  Will post it once I have one.  Thanks.
